.github/workflows/linting.yml

```diff
@@ -28,6 +28,9 @@ jobs:
       # Whether any JavaScript files have changed.
       js: ${{ steps.filter.outputs.js }}
 
+      # Whether any style files have changed.
+      css: ${{ steps.filter.outputs.css }}
+
       # Whether any lock files have changed.
       lockfiles: ${{ steps.filter.outputs.lockfiles }}
 
```

Comment:
You forgot to add an output like
```
      # Whether any miscellaneous files related to CSS linting have changed.
      misc_css: ${{ steps.filter.outputs.misc == 'true' || steps.filter.outputs.misc_css == 'true' }}
```
in here (a little below where this comment is)

Reply:
Oops: dcfca677a7